util/logger.js

```diff
@@ -0,0 +1,63 @@
+/* eslint no-underscore-dangle: [2, { "allow": ["_logInfo", "_logError", "_logDebug"] }] */
+/* eslint import/no-dynamic-require: 0 */
+const winston = require('winston');
+const fs = require('fs');
+const path = require('path');
+/* The environment variable LOGGERCONFIG will contain the path to the config file.
+  The actual path for the config is "../deploy/configs/util/logger.json"
+  For the docker containers, the path is /etc/util/logger.json */
```

Comment:
please add a check about the existence of a proper value in "process.env.LOGGERCONFIG" variable. If the variable is not properly initialized, we can throw a meaningful error and bail out.